src/lib/components/Form/FormHelpText.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import getClassName from '../../utils/getClassName';
+
+const FormHelpText = (props) => {
+  const { children, className } = props;
+
+  const classNames = getClassName({
+    'p-form-help-text': true,
+    [`${className}`]: className,
+  }) || undefined;
```

Comment:
Could we replace `undefined` with `{}`? This way if `classNames` doesn't exist for some reason the component won't error.